src/components/AttachmentsModal/AttachmentsModal.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import { Modal, Segment, Button, Icon } from 'semantic-ui-react'
+import { Component } from 'react'
```

Comment:
use `import * as React rom 'react` instead.
The component class should extends React.component.